.github/workflows/ci.yml

```diff
@@ -9,14 +9,15 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, "3.10", 3.11, pypy2.7, pypy3.9]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, "3.10", 3.11, 3.12, pypy2.7, pypy3.9]
 
     steps:
     - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435 # v4
+      uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4
```

Comment:
Comment needs updating to indicate what version the updated hash refers to.

Reply:
Done :smile: